model/project.go

```diff
@@ -958,7 +958,12 @@ func generateIdsForVariant(vt TVPair, proj *Project, v *Version, tasks TVPairSet
 	return table
 }
 
+// generateId generates a unique project ID. For tasks created for untracked branches,
+// use owner/repo/branch in lieu of a user-defined project identifier.
```

Comment:
Nit: owner_repo_branch

Reply:
I more meant to enumerate the fields we were using in place but I see why this isn't clear; i'll switch to owner, repo, and branch